lib/tasks/demo.js

```diff
@@ -199,8 +193,9 @@ function hasUniqueNames(demos) {
 
 function updateOrigamiJsonDemoList(demosBuilt) {
 	var origamiDemoConfig = demosBuilt.map(function(conf) {
+
 		return {
-			path: "/" + conf.outputHtmlPath,
+			path: '/demos/' + conf.name + '.html',
```

Comment:
Should really use `path.join` to build paths, it's more portable (handling special cases for windows), and it's safer.